portfolio/src/main/webapp/script.js

```diff
@@ -56,6 +56,20 @@ function handleData(data) {
 function addDataToDom(data) {
   const dataContainer = document.getElementById('data-container');
   
-  dataContainer.innerText = data;
-  console.log(data);
+  dataContainer.innerText = commentsList(splitComments(data));
+  console.log();
+}
+
+function splitComments(data) {
+    var comments = data.split(",");
+    return comments;
+}
```

Comment:
since this method basically just calls data.split, it probably clearer to just do that in the above line or inside the commentsList function rather than having a separate function for it unless you have later plans for that function. so you could have:

`dataContainer.innerText = commentsList(data.split(","));`

or something like:

```
function commentsList(data) {
    splitComments = data.split(",");
    var comments = "";
    for (i = 0; i < splitComments.length; i++) {
        comments += splitComments[i] + "\n";
        console.log(splitComments[i]);
    }
    return comments;
}
```